lib/Event/PostBuilderInitEvent.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\RST\Event;
+
+use Doctrine\Common\EventArgs;
+use Doctrine\RST\Builder;
+use Doctrine\RST\Configuration;
+
+/**
+ * This event is called after the Builders initialization is finished,
+ * before the actual build starts.
+ *
+ * It replaces the former method Kernel::initBuilder()
```

Comment:
This comment is useful for reviewers of this PR, but I think it should be removed from the source code, since `Kernel` itself is removed. If people outside the library need to know about this, it could go in `UPGRADE.md`, which is where it's fine to keep a bit of history.

Same for similar comments below.